packages/report-server/src/__tests__/reportBuilder/functions.test.ts

```diff
@@ -103,25 +102,24 @@ describe('functions', () => {
   });
 
   describe('math', () => {
-    // TODO: Fixup test after getting sum working correctly
-    // describe('sum', () => {
-    //   it('sums a list of numbers', () => expect(functions.sum(1, 2, 3)).toBe(6));
+    describe('sum', () => {
+      it('sums a list of numbers', () => expect(functions.sum(1, 2, 3)).toBe(6));
 
-    //   it('ignores undefined values', () => expect(functions.sum(1, undefined, 3)).toBe(4));
+      it('ignores undefined values', () => expect(functions.sum(1, undefined, 3)).toBe(4));
 
-    //   it('can sum an array', () => expect(functions.sum([1, 2, 3])).toBe(6));
+      it('can sum an array', () => expect(functions.sum([1, 2, 3])).toBe(6));
 
-    //   it('ignores undefined within an array', () =>
-    //     expect(functions.sum([1, undefined, 3])).toBe(4));
+      it('ignores undefined within an array', () =>
+        expect(functions.sum([1, undefined, 3])).toBe(4));
 
-    //   it('can combine numbers and arrays', () =>
-    //     expect(functions.sum([1, undefined, 3], undefined, 2)).toBe(6));
+      it('can combine numbers and arrays', () =>
+        expect(functions.sum([1, undefined, 3], undefined, 2)).toBe(6));
 
-    //   it('returns undefined if all values are undefined', () =>
-    //     expect(functions.sum([undefined, undefined, undefined], undefined, undefined)).toBe(
-    //       undefined,
-    //     ));
-    // });
+      it('returns undefined if all values are undefined', () =>
```

Comment:
Should this be `0` maybe? I wasn't too sure... I feel either could be seen as appropriate...

Reply:
I think undefined is the correct answer